SonarQube.Common/FilePropertiesProvider.cs

```diff
@@ -24,11 +24,6 @@ public class FilePropertiesProvider : ISonarPropertyProvider
         /// </summary>
         private IDictionary<string, string> properties;
 
-        /// <summary>
-        /// Property name comparisons are be case-sensitive
-        /// </summary>
-        private static StringComparer PropertyKeyComparer = StringComparer.InvariantCulture;
```

Comment:
The string comparer used to be InvariantCulture and it is now Ordinal. Since the properties are strings that users provide, shouldn't we be using the CurrentCulture instead?